src/editors/containers/TextEditor/TextEditor.test.jsx

```diff
@@ -1,51 +1,86 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
-import TextEditor from './TextEditor';
-import EditorPageContext from '../EditorPageContext';
-import { ActionStates } from '../data/constants';
+import { shallow } from 'enzyme';
+import { TextEditor, mapStateToProps, mapDispatchToProps } from './TextEditor';
+import { actions, selectors } from '../../data/redux';
+import { RequestKeys } from '../../data/constants/requests';
 
 // Per https://github.com/tinymce/tinymce-react/issues/91 React unit testing in JSDOM is not supported by tinymce.
 // Consequently, mock the Editor out.
-const mockRole = 'Tiny-MCE-Mock';
 jest.mock('@tinymce/tinymce-react', () => {
   const originalModule = jest.requireActual('@tinymce/tinymce-react');
   return {
     __esModule: true,
     ...originalModule,
-    Editor: () => <div role={mockRole} />
+    Editor: () => <div />
```

Comment:
I changed it to be a string, but since this is a shallow render, it shouldn't show up anyway. But if you do see that string in the shallow render, something has messed up, so thats good.